### DISCUSSION TITLE
Nuxt issue. How to use .env file to distinguish Environment Variables: develpoment environment and production environment

### DISCUSSION THREAD

**Discussion by zhizaiyunxiao** [nuxt/nuxt#20077] [Questions]
### Environment

![image](https://user-images.githubusercontent.com/22257246/229758611-d0cff321-5af2-44bf-a745-5402b1eb5962.png)
I don't understand this Nuxt document 

### Reproduction

Nuxt issue. How to use .env file to distinguish Environment Variables: develpoment environment and production environment

### Describe the bug

Nuxt issue. How to use .env file to distinguish Environment Variables: develpoment environment and production environment

**Comment by memic84** [comment]
Did you read this: https://nuxt.com/docs/guide/directory-structure/env ?

It all depends on your configuration / setup, can you provide more information?

**Reply by zhizaiyunxiao** [reply]
But I don't understand this document. Can you give a simple example or website?

**Reply by memic84** [reply]
What's your production environment? How do you deploy your site?

**Comment by liulib** [comment]
哥们，你咋解决的。。。
